### PR TITLE
Add The state fo Python in blockchain 2023 report to Python docs

### PR DESCRIPTION
**Pull request by miohtama**
I am adding a link to The state of Python in the blockchain 2023 report. It includes an overview of Python developer opportunities in the Ethereum ecosystem and lists ~15 different Python-based projects for Ethereum developer and integrators.

As a disclaimer I am one of the authors of the report.

[Here is the direct link to the report](https://tradingstrategy.ai/blog/the-state-of-python-in-blockchain-in-2023).

## Description

Adding a new link to the Beginner articles section on Python documentation

### CODE REVIEW

File: src/content/developers/docs/programming-languages/python/index.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Need a more basic primer first? Check out [ethereum.org/learn](/learn/) or [ethe
 - [Write your First Smart Contract](https://kauri.io/article/124b7db1d0cf4f47b414f8b13c9d66e2/remix-ide-your-first-smart-contract)
 - [Learn How to Compile and Deploy Solidity](https://kauri.io/article/973c5f54c4434bb1b0160cff8c695369/understanding-smart-contract-compilation-and-deployment)
 
+
```
Comment (minimalsm) [inline src/content/developers/docs/programming-languages/python/index.md:23]:
```suggestion
```

File: src/content/developers/docs/programming-languages/python/index.md [hunk 2]
```diff
@@ -28,6 +28,7 @@ Need a more basic primer first? Check out [ethereum.org/learn](/learn/) or [ethe
 - [How to develop Ethereum contract using Python Flask?](https://medium.com/coinmonks/how-to-develop-ethereum-contract-using-python-flask-9758fe65976e)
 - [Intro to Web3.py · Ethereum For Python Developers](https://www.dappuniversity.com/articles/web3-py-intro)
 - [How to call a Smart Contract function using Python and web3.py](https://stackoverflow.com/questions/57580702/how-to-call-a-smart-contract-function-using-python-and-web3-py)
+- [The state of Python in blockchain 2023 report features multiple Python integration, SDK and security tools for Ethereum](https://tradingstrategy.ai/blog/the-state-of-python-in-blockchain-in-2023)
```
Comment (minimalsm) [inline src/content/developers/docs/programming-languages/python/index.md:31]: I think we could remove the specifics here.

```suggestion
- [The state of Python in blockchain 2023 report](https://tradingstrategy.ai/blog/the-state-of-python-in-blockchain-in-2023)
```

Comment (minimalsm) [reply src/content/developers/docs/programming-languages/python/index.md:31]: Also, I'd probably move this higher up the list (maybe second in the list?)

**Review by minimalsm — COMMENTED** [review]
Thanks @miohtama, small change request, let me know if you disagree. Good with this going in either way ❤️

File: src/content/developers/docs/programming-languages/python/index.md (hunk 2 above)
Comment (corwintines) [reply src/content/developers/docs/programming-languages/python/index.md:31]: Will make these changes and merge

**Review by corwintines — APPROVED** [review]

**Pull request merged**